docs/sphinx/manual/building_hpx.rst

```diff
@@ -181,7 +59,7 @@ libraries, you need to specify them as link dependencies in your build (See
 .. _important_cmake_options:
 
 Most important |cmake| options 
-------------------------------
+==============================
 
 While building |hpx|, you are provided with multiple CMake options which correspond 
 to different configurations. Below, there is a set of the most important and frequently 
```

Comment:
Now that the full list of options is in a separate page, maybe it would be good to add a link to it here? Or do we already have a link somewhere nearby that I've missed?